src/lib/hooks/useAccountSwitcher.ts

```diff
@@ -44,9 +45,14 @@ export function useAccountSwitcher() {
             'circle-exclamation',
           )
         }
-      } catch (e) {
-        Toast.show('Sorry! We need you to enter your password.')
+      } catch (e: any) {
+        logger.error(`switch account: selectAccount failed`, {
+          message: e.message,
+        })
         clearCurrentAccount() // back user out to login
+        setTimeout(() => {
+          Toast.show('Sorry! We need you to enter your password.')
```

Comment:
Sorry to jump in here, would it be possible to mark this toast for localisation? And ideally the ones in L38 and L43-44 too? 🙏

Reply:
Great call, thanks for the reminder!